widgetastic_manageiq.py

```diff
@@ -1205,15 +1206,15 @@ class ItemsToolBarViewSelector(View):
         view_selector.selected
     """
     ROOT = './/div[contains(@class, "toolbar-pf-view-selector")]'
-    grid_button = Button(title='Grid View')
-    tile_button = Button(title='Tile View')
-    list_button = Button(title='List View')
+    btn_class = VersionPick({
```

Comment:
@izapolsk the Button class works for 5.7, I would be version picking all of the element interaction in the class that works for both - that was my first implementation but @mfalesni objected to any version picking in the widgetastic_patternfly module.